src/main/resources/assets/lambdabettergrass/lang/ru_ru.json

```
@@ -0,0 +1,12 @@
+{
+  "lambdabettergrass.menu.info.1": "Мод использует ресурспак, чтобы определить, какие блоки затронуты.",
+  "lambdabettergrass.menu.info.2": "Включение или отключение функции улучшенного дёрна/снега на блоках",
+  "lambdabettergrass.menu.info.3": "нужно модифицировать с помощью ресурспака.",
```

```suggestion
  "lambdabettergrass.menu.info.3": "необходимо производить с помощью пакетов ресурсов.",
```
Более близко к оригинальному контексту